Action.h

```diff
@@ -36,8 +36,8 @@ typedef struct State_ {
    Machine* host;
    struct MainPanel_* mainPanel;
    Header* header;
+   const char* failedUpdate; /* function bar diagnostic or NULL if no error */
```

Comment:
Rename this to `collectionStateDiagnostic`, which somewhat better aligns with its purpose. Also better hints at the possible use case of not just representing an error state …

Reply:
Yow, that's quite a mouthful! :)  I think I prefer the simpler, shorter name TBH, and the way it aligns with pausedUpdate on the line below.  The intention is to represent errors, not arbitrary messages (recall we're using FAILED_READ as the color here too), so I prefer if we leave this part as-is.

Reply:
Good point.